0004-Update-release-in-appdata.patch

```diff
@@ -0,0 +1,40 @@
+From b12084995d14195c7f72a0fa031b36ae456c09f9 Mon Sep 17 00:00:00 2001
+From: bbhtt <62639087+bbhtt@users.noreply.github.com>
+Date: Tue, 24 May 2022 07:13:17 +0530
+Subject: [PATCH] Update release in appdata and add width and height to image
+ tag
+
+---
+ com.github.davetheunsub.clamtk.appdata.xml | 9 +++++++--
+ 1 file changed, 7 insertions(+), 2 deletions(-)
+
+diff --git a/com.github.davetheunsub.clamtk.appdata.xml b/com.github.davetheunsub.clamtk.appdata.xml
+index c6d0713..2bb7739 100644
+--- a/com.github.davetheunsub.clamtk.appdata.xml
++++ b/com.github.davetheunsub.clamtk.appdata.xml
+@@ -79,15 +79,20 @@
+  <screenshots>
+      <screenshot type="default">
+          <caption>The initial screen</caption>
+-         <image>https://davem.fedorapeople.org/images/clamtk-1-screenshot.png</image>
++         <image width="504" height="502">https://davem.fedorapeople.org/images/clamtk-1-screenshot.png</image>
```

Comment:
not sure why you change this. width and height are not required.

Reply:
Yea, I'll remove that later if there aren't anymore fixes. I learnt that later. Thanks.